modules/distribution/src/repository/resources/dbscripts/apimgt/oracle.sql

```diff
@@ -195,7 +197,7 @@ CREATE SEQUENCE IDN_STS_STORE_SEQUENCE START WITH 1 INCREMENT BY 1 NOCACHE
 /
 
 CREATE OR REPLACE TRIGGER IDN_STS_STORE_TRIGGER
-            BEFORE INSERT
+		    BEFORE INSERT
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04